ursa_core/src/lib.rs

```diff
@@ -1 +1,25 @@
+//!
```

Comment:
Missing documentation. This will not be overridden. I suggest adding in a couple of lines or removing the doc marker entirely.